operate/client/src/App/Processes/MigrationView/TopPanel/Diagrams/v2/SourceDiagram.tsx

```diff
@@ -51,13 +50,13 @@ const SourceDiagram: React.FC = observer(() => {
     return 'content';
   };
 
-  const processInstanceFilters = useProcessInstanceFilters();
-
   const {data: overlayData} = useProcessInstancesOverlayData(
     {
-      ...processInstanceFilters,
-      processInstanceKey: getProcessInstanceKey(),
+      filter: {
+        processInstanceKey: getProcessInstanceKey(),
+      },
     },
+    sourceProcessDefinitionKey!,
```

Comment:
useProcessInstancesOverlayData accepts undefined for sourceProcessDefinitionKey

```suggestion
    sourceProcessDefinitionKey,
```

Reply:
Ah I didn't notice the bang functions 🤦, for some reasons `sourceProcessDefinitionKey` can be `string | null | undefined`. `undefined` can be handled by the hook but not `null`. I think it's better to keep the hook that way and fallback to `undefined` whenever `sourceProcessDefinitionKey` is null. This behavior will be removed along with the store later on.

I'm changing the code to `sourceProcessDefinitionKey ?? undefined`